app/routes/template/template.js

```diff
@@ -72,8 +72,6 @@ router.route('/:template([A-z0-9-]{36})')
           data: logo.data,
           filename: logo.name,
           format: logo.mimetype,
```

Comment:
I think the frontend might need the logo and header to have a specific height and width, @kttkjl can you confirm/deny?

Reply:
pinging @kttkjl

Reply:
as long as the aspect ratio isn't changed, we have an attribute on the front-end that limits the image size displayed 

so whatever the backend sends, shouldn't matter

Reply:
ok great, thank you!